packages/Path-View-Core.package/SClassDiagram.class/instance/setSemanticZooming..st

```diff
@@ -0,0 +1,11 @@
+zooming
+setSemanticZooming: aNumber
+
+	| fontSize |
+	self isSemanticZooming 
```

Comment:
you are absolutely right, but this check makes sure, i only traverse the diagram graph once to collapse all morphs. i renamed this to isAlreadySemanticZooming which should make things more clearly